src/lib/dnssd/Advertiser_ImplMinimalMdns.cpp

```diff
@@ -228,7 +228,7 @@ class AdvertiserMinMdns : public ServiceAdvertiser,
                     mrp.mIdleRetransTimeout = kMaxRetryInterval;
                 }
                 size_t writtenCharactersNumber = snprintf(storage.mrpRetryIntervalIdleBuf, sizeof(storage.mrpRetryIntervalIdleBuf),
-                                                          "CRI=%" PRIu32, mrp.mIdleRetransTimeout.count());
+                                                          "SII=%" PRIu32, mrp.mIdleRetransTimeout.count());
```

Comment:
I assume there will be a followup to rename mIdleRetransTimeout to something that more closely matches what this actually is?

Reply:
Yes, this PR is focused on cleanly renaming the Discovery names only to minimize impact on CSG.

Reply:
#17999